src/evict/evict_lru.c

```diff
@@ -447,16 +448,38 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
 
     WT_ASSERT_SPINLOCK_OWNED(session, &evict->evict_pass_lock);
 
+    evict->use_npos_in_pass = __wt_atomic_loadbool(&conn->evict_use_npos);
```

Comment:
Can we directly assign the config to evict->use_npos_in_pass? Is this to prevent racing with setting the cofig? If so, can we add a comment here.

Reply:
Added a comment